kubeflow/core/prototypes/all.jsonnet

```diff
@@ -16,6 +16,8 @@
 // @optionalParam jupyterNotebookPVCMount string null Mount path for PVC. Set empty to disable PVC
 // @optionalParam reportUsage string false Whether or not to report Kubeflow usage to kubeflow.org.
 // @optionalParam usageId string unknown_cluster Optional id to use when reporting usage to kubeflow.org
+// @optionalParam pytorchJobImage string jgaguirr/pytorch-operator:latest The image for the PyTorchJob controller
```

Comment:
@jlewi Is there any place I can release the latest image to?

Reply:
kubeflow-images-staging

Reply:
gcr.io/kubeflow-images-staging

Reply:
@jlewi How do I push there?

Reply:
The Kubeflow release team has permission to push there.

We should probably automate this.

Reply:
Our release process is described here:
https://github.com/kubeflow/kubeflow/blob/master/docs_dev/releasing.md

So the steps would be something like the following

1. Create an Argo workflow as an E2E test builds the image and runs some tests to validate it.
1. Add a section to the release guide about how to build and release the operator image
1. Request someone in the kubeflow-release team does this

We should automate this so we can push images on each post submit. 

We could also potentially make our CI images public. Not sure what the best approach would be.

Reply:
We have gcr.io/kubeflow-images-public now. So you could use that as the URI